tests/workflows/parallel_node_cancellation/tests/test_workflow.py

```diff
@@ -28,3 +28,10 @@ def test_parallel_node_cancellation__streaming():
     assert top_node_rejection.error.message == "Top node failed"
 
     assert bottom_node_rejection.error.code == WorkflowErrorCode.NODE_CANCELLED
+
+    last_event = events[-1]
+    assert (
+        last_event.name == "workflow.execution.rejected"
+    ), f"Expected last event to be workflow rejection, got {last_event.name}"
+    assert last_event.error.code == top_node_rejection.error.code
+    assert last_event.error.message == top_node_rejection.error.message
```

Comment:
DevinAI, this looks good. Can we additionally verify that the bottom node rejection is the second to last event, and that the top node rejection is the third to last event